.github/workflows/main.yml

```diff
@@ -33,6 +33,32 @@ jobs:
     - name: Make distcheck
       run: make distcheck
 
+  build-usan:
+    name: Build with UCS2 and usan
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: sudo apt-get update -qq && sudo apt-get install -y libyaml-dev texinfo texlive
```

Comment:
TBH, not sure if we really need `texlive`. I just tried inside a docker image and it seems to compile without `texlive`

Reply:
That was actually needed for distcheck, which we drop, so I have now dropped it, and texinfo as well